OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -631,6 +648,121 @@ public async Task<SearchResult<WorkshopCard>> GetNearestByFilter(WorkshopFilter
         return result;
     }
 
+    /// <inheritdoc/>
+    /// <exception cref="InvalidOperationException">If the region admin is not found in the database.</exception>
+    public async Task<SearchResult<WorkshopDto>> FetchByFilterForAdmins(WorkshopFilterAdministration filter = null)
+    {
+        logger.LogInformation("Started retrieving Workshops by filter for admins.");
+        filter = filter ?? new WorkshopFilterAdministration();
```

Comment:
could you please add a `Debug` log that value is null and we continue with the default (WorkshopFilterAdministration) filter

Reply:
fixed